schemas/io-package.json

```diff
@@ -1100,6 +1100,62 @@
           "minimum": 1,
           "maximum": 3,
           "default": 3
+        },
+        "messages": {
+          "description": "Messages can be shown on adapter installation, update and instance creation",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["title", "text", "level", "buttons"],
```

Comment:
What is really required?

Reply:
- level is optional as it has default value "warn"
- title is mandatory
- text is mandatory
- link is optional
- linkText is optional and has default value "More info"

Reply:
Buttons?

Reply:
Buttons are required too